### PR TITLE
Synonym value

### PR DESCRIPTION
**Pull request by prasanth55555**
**Proposed changes**:
- by using the change we can be able to pass the synonym value along wiith the original value

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by msamogh** [comment]
Thanks for the PR, we'll give it a review as soon as possible

File: rasa/nlu/extractors/entity_synonyms.py [hunk 1]
```diff
@@ -85,6 +85,7 @@ def replace_synonyms(self, entities):
             # need to wrap in `str` to handle e.g. entity values of type int
             entity_value = str(entity["value"])
             if entity_value.lower() in self.synonyms:
+                entity["synonym"] = entity_value.lower()
```
Comment (erohmensing) [inline rasa/nlu/extractors/entity_synonyms.py]: If you need the information of the original text before it gets replaced, do you actually want it to be lowercase?

Comment (erohmensing) [inline rasa/nlu/extractors/entity_synonyms.py]: If you need the information of the original text before it gets replaced, do you actually want it to always be stored lowercase?

**Comment by erohmensing** [comment]
@prasanth55555 still interested in merging this? I just had one question

**Comment by erohmensing** [comment]
closing as the PR is stale and the branch is diverging from the original PR

**Pull request closed without merging**